app/assets/stylesheets/application.css.scss

```diff
@@ -104,3 +104,8 @@ i.fa.medium {
   transform: translateX(-50%);
   z-index: 997;
 }
+
+// darkens text for input placeholders
+::-webkit-input-placeholder {
+    color: #565656 !important;
```

Comment:
Fixed in [6fba5b1](https://github.com/OperationCode/operationcode/pull/638/commits/6fba5b1524415e98158d24f03665566498a39507)

Woof!